Makefile

```diff
@@ -30,12 +30,20 @@ build: test-data
 
 ## test-data
 testDataDir := src/test/resources/
+tempDir := ${testDataDir}/temp
+gitDataDir := ${tempDir}/sdk-test-data
+branchName := main
+githubRepoLink := https://github.com/Eppo-exp/sdk-test-data.git
 .PHONY: test-data
 test-data: 
 	rm -rf $(testDataDir)
-	mkdir -p $(testDataDir)
-	gsutil cp gs://sdk-test-data/rac-experiments-v3.json $(testDataDir)
-	gsutil cp -r gs://sdk-test-data/assignment-v2 $(testDataDir)
+	mkdir -p $(tempDir)
+	cd ${tempDir} \
+	    && git clone -b ${branchName} --depth 1 --single-branch ${githubRepoLink} \
+	    && rm -rf RepoName/.git/
+	cp ${gitDataDir}/rac-experiments-v3.json ${testDataDir}
+	cp -r ${gitDataDir}/assignment-v2 ${testDataDir}
+	rm -rf ${tempDir}
```

Comment:
First I am cloning the git repo into a temp folder and then copying only those files which are required for the testing. This gives some freedom to the dependent repo to only pull those files which are required.